src/LibraryViewExtensionMSWebBrowser/web/library/layoutSpecs.json

```diff
@@ -1454,11 +1454,14 @@
                       "inclusive": false
                     },
                     {
-                      "path": "GeometryColor.Autodesk.DesignScript.Geometry.ImportHelpers.Geometry",
-                      "inclusive": false
+                      "path": "Geometry.Geometry.ExportToSAT"
                     },
                     {
-                      "path": "Geometry.Geometry.ExportToSAT"
+                      "path": "Geometry.Geometry.Geometry.ImportFromSatAndUnits"
```

Comment:
Why does the export node `Geometry.Geometry.Export`, whereas the new node is `Geometry.Geometry.Geometry...`

Reply:
Because the existing ExportToSAT node appears in the UI as `ExportToSAT` - but I thought we wanted this node to appear as `Geometry.ImportFromSATblahblahblah` - and I thought that, because the existing SAT import nodes look like: `Geometry.ImportFromSAT`
![Screen Shot 2022-08-22 at 12 43 56 PM](https://user-images.githubusercontent.com/508936/185974614-8e7dd28f-b509-4129-981a-2b23fe79efc7.png)
![Screen Shot 2022-08-22 at 12 43 50 PM](https://user-images.githubusercontent.com/508936/185974635-a261e701-5c82-401c-af7b-e2287a37fc97.png)

Reply:
Ok, great 👍